language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/providers/ExtractToFunctionCodeAction.java

```diff
@@ -490,6 +501,16 @@ private Optional<ArgListsHolder> getArgLists(CodeActionContext context, List<Sym
         return Optional.of(argListsHolder);
     }
 
+    private Optional<TypeSymbol> getTypeSymbol(CodeActionContext context, TypeSymbol typeSymbol,
```

Comment:
Let's change this name to something like getTypeForArgument()

Reply:
+1